apps/op-func/src/infra/storage/access-log-register.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  BlobServiceClient,
+  BlockBlobUploadResponse,
+  ContainerClient,
+} from "@azure/storage-blob";
+
+interface AccessLogRecord {
+  fiscalCode: string;
+  idToken: string;
+  ipAddress: string;
+  rpAuthRequest: string;
+  timestamp: string;
+}
+
+export class AccessLogRegister {
+  #container: ContainerClient;
+
+  constructor(blobServiceClient: BlobServiceClient, containerName: string) {
+    this.#container = blobServiceClient.getContainerClient(containerName);
+  }
+
+  // A helper method used to read a Node.js readable stream into a String
+  async #streamToBuffer(readableStream: NodeJS.ReadableStream) {
+    return new Promise<AccessLogRecord>((resolve, reject) => {
+      const chunks: Buffer[] | Uint8Array[] = [];
+      readableStream.on("data", (data) => {
+        chunks.push(data instanceof Buffer ? data : Buffer.from(data));
+      });
+      readableStream.on("end", () => {
+        resolve(JSON.parse(Buffer.concat(chunks).toString()));
+      });
+      readableStream.on("error", reject);
+    });
+  }
+
+  async get(name: string): Promise<AccessLogRecord> {
+    const blobClient = this.#container.getBlobClient(name);
+
+    // Get blob content from position 0 to the end
+    // get downloaded data by accessing downloadBlockBlobResponse.readableStreamBody
+    const downloadBlockBlobResponse = await blobClient.download();
+    const downloadedStream = downloadBlockBlobResponse.readableStreamBody;
+    if (!downloadedStream)
+      throw new Error("No Blob with name " + name + " found");
```

Comment:
here you can use `assert.ok()` from `node:assert/strict` instead of `if + throw`